src/Cronofy/Responses/AddToCalendarResponse.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Cronofy.Responses
+{
+    using System;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Class for the deserialization of an add to calendar response.
+    /// </summary>
+    internal sealed class AddToCalendarResponse
+    {
+        /// <summary>
+        /// Gets or sets the OAuthUrl of the response.
+        /// </summary>
+        /// <value>
+        /// The OAuth URL for the user to visit in order to add the event to their calendar.
+        /// </value>
+        [JsonProperty("oauth_url")]
```

Comment:
Noticed this is correct. This should be more opaque in name. `oauth_url` is too descriptive of the underlying URL in my opinion. It might not always be an OAuth URL for example and then this name would be weird.

If it were just `url` or `add_to_calendar_url` then we could change where it actually redirects without it feeling odd.